src/array.js

```diff
@@ -111,20 +111,24 @@ TransisArray.from = function(a, mapFn, thisArg) {
     if (i === 0) { this.didChange('first'); }
     this.didChange('@');
 
-    if (this.__proxy__) {
-      removed.forEach(function(x) {
-        if (x instanceof TransisObject || x instanceof TransisArray) {
-          x._deregisterProxy(this.__proxy__.to, this.__proxy__.name);
-        }
-      }, this);
-
-      added.forEach(function(x) {
-        if (x instanceof TransisObject || x instanceof TransisArray) {
-          x._registerProxy(this.__proxy__.to, this.__proxy__.name);
-        }
-      }, this);
-
-      this.__proxy__.to.didChange(this.__proxy__.name);
+    if (this.__proxies__) {
+      for (let k in this.__proxies__) {
+        let {object, name} = this.__proxies__[k];
```

Comment:
Both of these could be `const` instead of `let`.

Reply:
Thanks, updated in 886d0fc.